YouTube sample incorrectly formatted at head.

P1 since it breaks the CI checks

```
Error:  src/main/java/com/example/json/YouTubeSample.java:[8] (header) RegexpHeader: Line does not match expected header line of '^\W+\*?\W*http://www.apache.org/licenses/LICENSE-2.0$'.
Error:  src/main/java/com/example/json/YouTubeSample.java:[16,1] (whitespace) EmptyLineSeparator: 'package' should be separated from previous statement.
Error:  src/main/java/com/example/json/YouTubeSample.java:[21,1] (imports) CustomImportOrder: Extra separation in import group before 'java.io.IOException'
Error:  src/test/java/com/example/json/YouTubeSampleTest.java:[8] (header) RegexpHeader: Line does not match expected header line of '^\W+\*?\W*http://www.apache.org/licenses/LICENSE-2.0$'.
Error:  src/test/java/com/example/json/YouTubeSampleTest.java:[16,1] (whitespace) EmptyLineSeparator: 'package' should be separated from previous statement.
Error:  src/test/java/com/example/json/YouTubeSampleTest.java:[33,1] (imports) CustomImportOrder: Extra separation in import group before 'java.io.IOException'
Error:  src/test/java/com/example/json/YouTubeSampleTest.java:[33,1] (imports) CustomImportOrder: Wrong lexicographical order for 'java.io.IOException' import. Should be before 'org.junit.Test'.
Error:  src/test/java/com/example/json/YouTubeSampleTest.java:[34,1] (imports) CustomImportOrder: Wrong lexicographical order for 'java.io.InputStream' import. Should be before 'org.junit.Test'.
Error:  src/test/java/com/example/json/YouTubeSampleTest.java:[35,1] (imports) CustomImportOrder: Wrong lexicographical order for 'java.util.Map' import. Should be before 'org.junit.Test'.
Error:  src/test/java/com/example/json/YouTubeSampleTest.java:[37,1] (imports) CustomImportOrder: Import statement for 'org.junit.Assert.assertEquals' is in the wrong order. Should be in the 'STATIC' group, expecting not assigned imports on this line.
Error:  src/test/java/com/example/json/YouTubeSampleTest.java:[38,1] (imports) CustomImportOrder: Import statement for 'org.junit.Assert.assertNotNull' is in the wrong order. Should be in the 'STATIC' group, expecting not assigned imports on this line.
Error:  src/test/java/com/example/json/YouTubeSampleTest.java:[44] (sizes) LineLength: Line is longer than 100 characters (found 104).
Error:  Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.1:check (default-cli) on project google-http-client-snippets: You have 12 Checkstyle violations. -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Error: Process completed with exit code 1.
```

--- a/samples/snippets/pom.xml
+++ b/samples/snippets/pom.xml
@@ -15,6 +15,7 @@
     <groupId>com.google.cloud.samples</groupId>
     <artifactId>shared-configuration</artifactId>
     <version>1.0.21</version>
+    <relativePath></relativePath>
   </parent>
 
   <properties>
@@ -58,12 +59,6 @@
       <groupId>com.google.truth</groupId>
       <artifactId>truth</artifactId>
       <version>1.1.2</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>com.google.http-client</groupId>
-      <artifactId>google-http-client-gson</artifactId>
-      <version>1.39.0</version>
       <scope>test</scope>
     </dependency>
     <!-- END_EXCLUDE -->

--- a/samples/snippets/src/main/java/com/example/json/YouTubeSample.java
+++ b/samples/snippets/src/main/java/com/example/json/YouTubeSample.java
@@ -5,7 +5,7 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *    https://www.apache.org/licenses/LICENSE-2.0
+ *    http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,

--- a/samples/snippets/src/main/java/com/example/json/YouTubeSample.java
+++ b/samples/snippets/src/main/java/com/example/json/YouTubeSample.java
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package com.example.json;
 
 import com.google.api.client.http.HttpResponse;

--- a/samples/snippets/src/test/java/com/example/json/YouTubeSampleTest.java
+++ b/samples/snippets/src/test/java/com/example/json/YouTubeSampleTest.java
@@ -5,7 +5,7 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *    https://www.apache.org/licenses/LICENSE-2.0
+ *    http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,

--- a/samples/snippets/src/test/java/com/example/json/YouTubeSampleTest.java
+++ b/samples/snippets/src/test/java/com/example/json/YouTubeSampleTest.java
@@ -13,8 +13,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package com.example.json;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import org.junit.Test;
 import com.google.api.client.http.HttpRequest;
 import com.google.api.client.http.HttpResponse;
 import com.google.api.client.http.HttpTransport;
@@ -28,14 +36,6 @@ import com.google.api.client.testing.http.MockHttpTransport;
 import com.google.api.client.testing.http.MockLowLevelHttpRequest;
 import com.google.api.client.testing.http.MockLowLevelHttpResponse;
 import com.google.common.base.Preconditions;
-import org.junit.Test;
-
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.Map;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
 
 public class YouTubeSampleTest {
 

